src/components/loading-modal/loading-modal.tsx

```diff
@@ -23,6 +23,21 @@ const mapStateToProps = (state: IReduxState) => {
 export const LoadingModalComponent = (
     props: IReduxProps & IExternalProps & IThemeProps<ReturnType<typeof stylesProvider>>
 ) => {
+    if (Platform.OS === 'web') {
```

Comment:
so `Modal` component from `react-native` is not working well on web ?

Reply:
![image](https://user-images.githubusercontent.com/32679726/75588347-1fdbc900-5a81-11ea-8de1-65d118968339.png)

`Modal` it's not working on web

Reply:
I made a little research, it seems that `modal` it's not supported yet on web Platform.

But instead this could be a quick fix if we want to use Modal on web (but I have not tested it, yet):
https://github.com/Dekoruma/react-native-web-modal

Reply:
yeah true, i think it's better :)
we can use https://github.com/Dekoruma/react-native-web-modal/tree/master/packages/modal-enhanced-react-native-web from there.